lib/features/modeling/Modeling.js

```diff
@@ -466,6 +468,18 @@ Modeling.prototype.updateWaypoints = function(connection, newWaypoints, hints) {
   this._commandStack.execute('connection.updateWaypoints', context);
 };
 
+Modeling.prototype.reconnect = function(connection, source, target, dockingOrPoints, hints) {
+  var context = {
+    connection: connection,
+    newSource: source,
+    newTarget: target,
+    dockingOrPoints: dockingOrPoints,
+    hints: hints || {}
+  };
+
+  this._commandStack.execute('connection.reconnect', context);
+};
+
 Modeling.prototype.reconnectStart = function(connection, newSource, dockingOrPoints, hints) {
```

Comment:
Please consider to remove (or point to Modeling#reconnect, if possible).

Same applies to `Modeling#reconnectEnd`.